tv_apps/smart-system/test/marionette/browser_context_menu_test.js

```diff
@@ -78,9 +78,11 @@ marionette.skip('Test Context Menu Events', function() {
   });
 
   test('press enter on second menu', { 'devices': ['tv'] }, function() {
-    launchContextMenu();
-    client.switchToFrame();
+    // Long press on a link
+    var link = client.helper.waitForElement('#link');
+    actions.longPress(link, 1.5).perform();
 
+    client.switchToFrame();
```

Comment:
Looks like we don't need this line. We'll change to System frame in the `system.waitForEvent`[1] method.
[1]: https://github.com/mozilla-b2g/gaia/blob/master/tv_apps/smart-system/test/marionette/lib/system.js#L64